downloads.html

```diff
@@ -80,13 +80,13 @@ <h2>Android</h2>
         <ul>
           <li>
             ARM:
-            <a href="https://github.com/minetest/minetest/releases/download/5.7.0/app-arm64-v8a-release.apk">64-bit</a> -
-            <a href="https://github.com/minetest/minetest/releases/download/5.7.0/app-armeabi-v7a-release.apk">32-bit</a>
+            <a href="https://github.com/minetest/minetest/releases/download/5.9.1/minetest-5.9.1-arm64-v8a-release.apk">64-bit</a> -
```

Comment:
Dead link? Looks like it should be https://github.com/minetest/minetest/releases/download/5.9.1/minetest-5.9.1-arm64-v8a.apk (minus the `release` part).